### DISCUSSION TITLE
Subsections on Organization and UBO

### DISCUSSION THREAD

**Discussion by MiraRatajczak** [OP-TED/eForms-SDK#374] [Q&A] [answered]
We have noticed in our application that the nodes ND-organization and ND-UBO have a group subsection above them. With the meta driven approach it creates 2 titles / sections with the same text. This does not seem consistent with the structure in other parts of the notice, e.g. node ND-Lot, which has no subsection above it. Can these subsections (“GR-Organisations-Subsection”, "GR-UBO-Subsection") be removed?

The sections highlighted in yellow seem to be superfluous:
![Subsections](https://user-images.githubusercontent.com/40240070/225315897-83cb84d0-c22a-4b8e-b7be-890ee8631f11.png)
![ND-Lot](https://user-images.githubusercontent.com/40240070/225315892-f20ec8b0-4b65-4451-b2f5-6485e49d2763.png)

**Comment by rouschr** [comment, marked as answer]
I think this has been solved by SDK 1.7
